test/test_meta.py

```diff
@@ -1086,6 +1097,20 @@ def __torch_dispatch__(self, func, types, args=(), kwargs=None):
 
         return expected
 
+
+def _handle_foreach_kwargs(func, args, kwargs):
+    if 'zero_size' in kwargs:
+        kwargs.pop('zero_size')
+    if 'disable_fastpath' in kwargs:
+        kwargs.pop('disable_fastpath')
```

Comment:
Why are these sampled in the first place?

Reply:
I'm not sure about the `zero_size` kwarg. Removing it still makes the tests pass.

About `disable_fastpath`, it's a keyword argument for testing with a fastpath and is not a keyword argument to the operation itself. I guess we could have a separate op_db with a value of `with_fastpath=True` and use that in tests that uses the fastpath argument.